README.md

```diff
@@ -1,14 +1,54 @@
 # The A11y Machine
 
-**The A11y Machine** is an **automated accessibility testing tool** which
-**crawls** and **tests** all pages of any web application. It validates pages
-against the following specifications/laws:
+**The A11y Machine** (or `a11ym` for short, spelled “alym”) is an **automated
+accessibility testing tool** which **crawls** and **tests** all pages of any web
+application, to finally produce detailed and efficient reports. It validates
```

Comment:
'finally' is not needed IHMO, it only make the sentence "heavier". Not sure about the 'efficient' also, I don't like bragging ^^ and what is an 'efficient report' anyway ?